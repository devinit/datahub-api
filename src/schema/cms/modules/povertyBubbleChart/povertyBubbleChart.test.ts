import {getPovertyBubbleChartPageData} from '.';
import * as prettyFormat from 'pretty-format';

describe('Poverty bubble chart module tests', () => {
    it('should return page data for the Poverty bubble chart', async () => {
        const pageData: DH.IPage[] = await getPovertyBubbleChartPageData();
      //  console.log(pageData[0]);
        expect(pageData.length).toBeGreaterThan(0);
        expect(prettyFormat(pageData)).toMatchSnapshot();
    }, );
});
