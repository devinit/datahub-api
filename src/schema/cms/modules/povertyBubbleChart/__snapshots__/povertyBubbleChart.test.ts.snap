// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Poverty bubble chart module tests should return page data for the Poverty bubble chart 1`] = `
"Array [
  Object {
    \\"id\\": \\"resources-able-to-meet-needs\\",
    \\"meta\\": \\"\\",
    \\"narrative\\": \\"The proportion of people living in extreme poverty worldwide has more than halved since 2000, yet progress is varied across countries. Although domestic public resources are an important resource to drive efforts to end poverty, they remain scarce in many countries where poverty is high\\",
    \\"title\\": \\"Are domestic public resources able to meet the needs of the poorest people?\\",
  },
]"
`;
